rebac-admin-backend/v1/interfaces/identities.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import r "github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+
+// IdentitiesService defines an abstract backend to handle Identities related operations.
+type IdentitiesService interface {
+
+	// ListIdentities returns a page of Identity objects of at least `size` elements if available
+	ListIdentities(params *r.GetIdentitiesParams) (*r.Identities, error)
```

Comment:
One important thing that we missed, is to have a `context.Context` parameter for the methods defined in the interface.

Reply:
Yes, I took a note about this to discuss with you, but I'm happy to put it here in next review, coming up